app/controllers/application_controller.rb

```diff
@@ -63,4 +63,10 @@ def prompt_to_set_up_2fa
   def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
+
+  def prompt_to_confirm_mobile(mobile, mobile_taken = false)
+    user_session[:unconfirmed_mobile] = mobile
+    user_session[:unconfirmed_mobile_taken] = mobile_taken
+    redirect_to phone_confirmation_new_path
+  end
```

Comment:
When calling this method from a controller and setting the second parameter to `true`, it's not clear what the second parameter represents. One would have to look up the method definition to figure that out. Instead, I recommend using a [keyword argument](https://robots.thoughtbot.com/ruby-2-keyword-arguments), like this:
```ruby
def prompt_to_confirm_mobile(mobile, mobile_taken: false)
  ...
end
```
Then in the controller, you can call it like this:
```ruby
prompt_to_confirm_mobile(@two_factor_setup_form.mobile, mobile_taken: true)
```
That's more readable.